.travis.yml

```diff
@@ -1,9 +1,24 @@
 language: python
+cache: pip
+env:
+  - PIN_REQUIREMENTS=1
+  - PIN_REQUIREMENTS=0
 python:
-    - "3.6"
+  - "3.6"
+matrix:
+  # Failures here don't mark the build as failed
+  allow_failures:
+    - env: PIN_REQUIREMENTS=0
```

Comment:
Largely because there's a lower priority to fix an issue if a new dependency release causes breakage. You don't want developers to be prevented merging a PR just because of a new third-party release. Instead, they should create an issue to support the new version, and implement that support in a separate PR when doing so becomes relevant.

Reply:
Ok! I've never used this before so out of curiousity: How do these allowed failures get reported? Do you still get an email but the test is just marked as a pass in github?

Reply:
I'm guessing so, but it's a while since it happened to me so I can't remember!